docs/Administration.rst

```diff
@@ -3061,6 +3064,136 @@ this entry is updated:
+-  a table with the whole applications list with delete and edit actions for each line (i.e. each application)
+-  ``Add an application`` button
+
+In this table, the three radio button properties: ``Mandatory``, ``Active`` and ``Mobile``,  
+are editable through the table. You just need to switch the radio button.
+
+.. _EditApp:
+
+Edit action
+------------
+
+When you click on the edit action, a drawer opens containing the already seized properties.
+
+|AC_image12|
+
+
+.. note:: For system applications, i.e. those installed in the platform, name, URL and image fields
+          are not editable. They appear grey colored in the edit mode.
+          
+          |AC_image13|
+          
+.. _DeleteApp:
+
+Delete action
+---------------
+
+When you click on the delete action, a popup appears to confirm the application's deletion.
+
+|AC_image14|
+
+Clicking on ``Delete`` button confirms the deletions, clicking on ``Cancel`` cancels teh deletion and nothing happens.
+
+.. _AddApp:
+
+Add an application
+~~~~~~~~~~~~~~~~~~~~
+
+In the :ref:`Applications tab <ApplicationsTabAppCenter>`, you have a button "Add new application" |AC_image7|
+allowing you to open the "Add application" drawer with these fields:
+
+|AC_image9|
+
+-  ``Title``: a mandatory field allowing to define the application's name
+-  ``URL``: a mandatory field allowing to set the access URL of the application
+-  ``Image``: not mandatory, allows you to define an icon for the application, you can browse for it through your files, it
+   should have a maximum size of 100 ko, otherwise, it will not be accepted.
+-  ``Description``: not mandatory, allows you to set a description for your application to explain more its usage. 
+   This field can contain a maximum of 1000 characters.
+-  ``Mandatory``: Allows you to define if the :ref:`application is mandatory <mandatoryApps>` for end users or not, 
+   by default set to NO   
+-  ``Active``: Allows you to define if the application should be enabled or not i.e. displayed to its 
+   audience (defined in the ``Permissions`` field), by default set to "YES" i.e. active
+-  ``Mobile``: Allows you to enable or disbale the application's display in mobile devices, i.e. if you judge it a not
+   mobile friendly app, you can set this option to false and that way ot will not be dispaled in mobile devices. 
+   By default set to true.
+-  ``Permissions``: to define the application's audience, By default set to *:/platform users i.e. if you don't define
+   this field, the added application will be displayed for the platform users
+-  ``Help page``: Allows you to set a help page URL, it could be any page containing more details about the application
+   or a how to guide. If you define this field, a :ref:`clickable question mark <helpPage>` will be added to the 
+   application's card and redirecting to the defined help page.
+   
+.. note:: The |AC_image8| button remains on disabled status until mandatory fields (``Title`` and ``URL``) are set.
```

Comment:
AC_image8 is missing and broken in the preview

Reply:
ok